src/features/events/components/EventOverviewCard/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+              color={getParticipantStatus()}
+              outlined={true}
+              size="sm"
+              value={`${availParticipants}/${reqParticipants}`}
+            />
+            <Box ml={1}>
+              <Settings
+                color="secondary"
+                cursor="pointer"
+                onClick={(event) =>
+                  setAnchorEl(anchorEl ? null : event.currentTarget)
+                }
+              />
+            </Box>
+            <Popper anchorEl={anchorEl} open={!!anchorEl}>
+              <ClickAwayListener
+                onClickAway={() => {
+                  setAnchorEl(null);
+                  if (
+                    newReqParticipants != null &&
+                    newReqParticipants != reqParticipants
+                  ) {
+                    model.setReqParticipants(newReqParticipants);
+                  }
+                }}
+              >
+                <Paper elevation={3} variant="elevation">
+                  <Box mt={1} p={2}>
+                    <TextField
+                      helperText={messages.eventParticipantsCard.reqParticipantsHelperText()}
+                      label={messages.eventParticipantsCard.reqParticipantsLabel()}
+                      onChange={(ev) => {
+                        const val = ev.target.value;
+
+                        if (val == '') {
+                          setNewReqParticipants(null);
+                          return;
+                        }
+
+                        const intVal = parseInt(val);
+                        if (!isNaN(intVal) && intVal.toString() == val) {
+                          setNewReqParticipants(intVal);
+                        }
+                      }}
+                      onKeyDown={(ev) => {
+                        if (ev.key === 'Enter') {
+                          setAnchorEl(null);
+                          if (newReqParticipants != null) {
+                            model.setReqParticipants(newReqParticipants);
+                          }
+                        } else if (ev.key === 'Escape') {
+                          setAnchorEl(null);
+                          setNewReqParticipants(reqParticipants);
+                        }
+                      }}
+                      value={
+                        newReqParticipants === null ? '' : newReqParticipants
+                      }
+                      variant="outlined"
+                    />
+                  </Box>
+                </Paper>
+              </ClickAwayListener>
+            </Popper>
+          </Box>
+        }
+      >
+        <Divider />
+        <Box
+          display="flex"
+          flexDirection="column"
+          justifyContent="space-between"
+        >
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginY={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
```

Comment:
This is minor, but you don't need to use theme to access the "secondary" text color! You can write it just as color='secondary' and MUI will know what you mean :)